Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.yml

```diff
@@ -9,18 +9,15 @@ configuration:
   required: true
   type: 0
 - display: API Token
-  name: apitoken
+  name: credentials
   required: true
-  type: 4
-- display: API Key
-  name: apikey
-  required: true
-  type: 4
-- defaultvalue: 3 days
-  display: First fetch time
+  type: 9
+  displaypassword: API Secret
+- display: First fetch time
```

Comment:
@NicCas There is a lot of reordering that happened, could you try reordering it so the diff goes away?